### DISCUSSION TITLE
PCL2在新建文件夹到dev0.0.1用了多久

### DISCUSSION THREAD

**Discussion by mc-124** [Meloong-Git/PCL#3345] [讨论]
.

**Comment by allMagicNB** [comment]
~~不不不第一个版本是Dev 0.0.0~~

**Comment by WForst-Breeze** [comment]
大概一两天

2018.8.7 透露说有了一丁点进展

2018.8.8 发布 dev 0.0.0，只有复选框

过了一会后

dev 0.0.1 就做出来了，有了按钮等三个控件

**Reply by allMagicNB** [reply]
不是，它之前就有考虑立项了

**Reply by allMagicNB** [reply]
得看看PCL1啥时候弃用……

**Reply by WForst-Breeze** [reply]
真正从开始做算起，周期应该就是很短的吧（

**Comment by mc-124** [comment]
改一下标题 dev0.0.0

**Comment by mc-124** [comment]
如果谁有保留到远古PCL2版本 发来看看？

**Reply by WForst-Breeze** [reply]
[Dev 0.0.0-0.0.15.zip](https://github.com/Hex-Dragon/PCL2/files/14318211/Dev.0.0.0-0.0.15.zip)

其实还有挺多，一直到 0.7.12 都是早期版本，但我就先给这么多玩玩得了🤔

**Reply by mc-124** [reply]
能不能给个0.4.4和0.7.12

**Reply by allMagicNB** [reply]
https://pan.baidu.com/s/1uHpP5KBsQZh8VxkYjxJqHQ?pwd=e4aa
https://pan.baidu.com/s/1ifcSpcybM0Ma2mZC7QIFqg?pwd=9ci4

**Comment by mc-124** [comment]
又开又关？

**Reply by allMagicNB** [reply]
关错了（
实际上并没有解决，而是超时

**Reply by Ad-closeNN** [reply]
关了的意义是啥，龙猫叫整理的吗（

**Reply by allMagicNB** [reply]
对啊，@WForst-Breeze 为啥你要关（

**Reply by WForst-Breeze** [reply]
？关什么，我没关
![image](https://github.com/Hex-Dragon/PCL2/assets/110760354/c9188856-1759-47fe-94ec-3df0411cecb4)

**Reply by allMagicNB** [reply]
> ？关什么，我没关
> ![image](https://github.com/Hex-Dragon/PCL2/assets/110760354/c9188856-1759-47fe-94ec-3df0411cecb4)
> 

不是，我是指其他 Discussion 你为啥要关掉超时的（

**Reply by WForst-Breeze** [reply]
> 不是，我是指其他 Discussion 你为啥要关掉超时的（

我只关提问板块的一直没有得到回答的和作者没动静的 discussion

**Reply by WorldHim** [reply]
> 关了的意义是啥，龙猫叫整理的吗（

记得有说过
大意是没有什么参考价值或者超时的就可以关掉

**Comment by Ad-closeNN** [comment]
?
